src/frontend/apps/standalone_site/src/features/Authentication/components/Authenticator.tsx

```diff
@@ -0,0 +1,74 @@
+import { Loader, useCurrentUser, useJwt } from 'lib-components';
+import { Fragment, PropsWithChildren, useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+
+import { getCurrentUser } from '../api/getUserData';
+import { validateChallenge } from '../api/validateChallenge';
+
+const TARGET_URL_STORAGE_KEY = 'redirect_uri';
+
+export const Authenticator = ({ children }: PropsWithChildren<unknown>) => {
```

Comment:
I would have exported from the index.ts file which behaves as the public API of the feature.

```suggestion
const Authenticator = ({ children }: PropsWithChildren<unknown>) => {
...
export default Authenticator;
```

Then in the index.ts at the feature root:
```
export { default as Authenticator} from './components/Authenticator';
```